File: VB/Custom Radial Menu/MyRadialMenuWindow.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraBars.Ribbon
Imports System.Windows.Forms
Imports DevExpress.Utils

Namespace WindowsFormsApplication4
    Public Class MyRadialMenuWindow
        Inherits RadialMenuWindow

        Public Sub New(ByVal menu As RadialMenu)
            MyBase.New(menu)
        End Sub
        Protected Overrides Sub OnMouseMove(ByVal e As MouseEventArgs)
            MyBase.OnMouseMove(e)
            Dim hitInfo As RadialMenuHitInfo = ViewInfo.CalcHitInfo(e.Location)
            Console.WriteLine(hitInfo.HitTest.ToString())
            Select Case hitInfo.HitTest
                Case RadialMenuHitTest.None
                    ToolTipController.DefaultController.HideHint()
                Case RadialMenuHitTest.Glyph
                    ShowHint("Glyph")
                Case RadialMenuHitTest.Link
                    ShowHint(hitInfo.LinkInfo.Link.Caption)
                Case RadialMenuHitTest.LinkArrow
                    ShowHint("LinkArrow")
            End Select
        End Sub
        Private Shared Sub ShowHint(ByVal text As String)
            ToolTipController.DefaultController.ShowHint(text, "Hovered element", Control.MousePosition)
        End Sub
    End Class
End Namespace

File: CS/Custom Radial Menu/MyRadialMenuWindow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraBars.Ribbon;
using System.Windows.Forms;
using DevExpress.Utils;

namespace WindowsFormsApplication4 {
    public class MyRadialMenuWindow : RadialMenuWindow {
        public MyRadialMenuWindow(RadialMenu menu)
            : base(menu) {
        }
        protected override void OnMouseMove(MouseEventArgs e) {
            base.OnMouseMove(e);
            RadialMenuHitInfo hitInfo = ViewInfo.CalcHitInfo(e.Location);
            Console.WriteLine(hitInfo.HitTest.ToString());
            switch (hitInfo.HitTest) {
                case RadialMenuHitTest.None:
                    ToolTipController.DefaultController.HideHint();
                    break;
                case RadialMenuHitTest.Glyph:
                    ShowHint("Glyph");
                    break;
                case RadialMenuHitTest.Link:
                    ShowHint(hitInfo.LinkInfo.Link.Caption);
                    break;
                case RadialMenuHitTest.LinkArrow:
                    ShowHint("LinkArrow");
                    break;
            }
        }
        private static void ShowHint(string text) {
            ToolTipController.DefaultController.ShowHint(text, "Hovered element", Control.MousePosition);
        }
    }
}
